opencog/combo/converter/combo_atomese.cc

```diff
@@ -125,6 +129,12 @@ std::pair<Type, Handle> vertex_2_atom::operator()(const builtin &b) const
 	return std::make_pair(type, Handle());
 }
 
+std::pair<Type, Handle> vertex_2_atom::operator()(const contin_t& c) const
+{
+	Handle h = createNode(NUMBER_NODE,to_string(c));
```

Comment:
@behailu04 , I think you can save the string conversion if you use `createNumberNode`.